content/en-us/reference/engine/classes/UnreliableRemoteEvent.yaml

```diff
@@ -28,18 +76,41 @@ methods:
     security: None
     thread_safety: Unsafe
   - name: UnreliableRemoteEvent:FireClient
-    summary: ''
-    description: ''
+    summary: |
+      Fires the `Class.UnreliableRemoteEvent.OnClientEvent|OnClientEvent` event for a
+      specific client connected to the same `Class.UnreliableRemoteEvent`. Has a 900 byte limit to the payload of the event, otherwise event is dropped.
```

Comment:
```suggestion
      Fires the `Class.UnreliableRemoteEvent.OnClientEvent|OnClientEvent` event for a
      specific client connected to the same `Class.UnreliableRemoteEvent`. Has a 900
      byte limit to the payload of the event, otherwise event is dropped.
```